### PR TITLE
Made form component

### PR DESCRIPTION
**Pull request by poojavishnoi**
Ignore absence of code snippets.I'll add them later

### CODE REVIEW

File: components/Form/form.css [hunk 1]
```diff
@@ -0,0 +1,119 @@
+:root {
+  --primary-color: #ff6d6d;
+  --text-color: rgb(73, 73, 73);
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
```
Comment (yashpatel1797) [inline components/Form/form.css:11]: Generally, we add this file in reset CSS so no need to write in every CSS file. just write once and import to another folder.

File: components/Form/form.css [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+
+.form_label,
+.form_input {
+  color: var(--text-color);
+  padding: 1rem 0.5rem;
+}
+
+.form_input {
+  border-radius: 5px;
+  border: 1px solid rgb(179, 179, 179);
```
Comment (yashpatel1797) [inline components/Form/form.css:34]: as you have used var everywhere to maintain consistency you can make changes here.

**Review by yashpatel1797 — COMMENTED** [review]
form design is clean.

File: components/Form/form.css (hunk 1 above)
Comment (poojavishnoi) [reply components/Form/form.css:11]: Yes. I have added this by mistake

File: components/Form/form.css (hunk 2 above)
Comment (poojavishnoi) [reply components/Form/form.css:34]: Okay

File: components/Form/form.css [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+  box-sizing: border-box;
+}
+
+.form_container {
+  margin: 1rem;
+}
+
+.form_main_container {
+  display: flex;
+  margin: 2rem 0rem;
```
Comment (ravikumar0403) [inline components/Form/form.css:19]: If the length is 0 then the unit (rem,px etc) can be skipped.

Comment (poojavishnoi) [reply components/Form/form.css:19]: Okay

**Pull request merged**